Wrappers/Python/cil/optimisation/algorithms/FISTA.py

```diff
@@ -121,8 +127,20 @@ def set_up(self, initial, f, g, step_size, **kwargs):
         # set up ISTA      
         self.initial = initial
         self.x_old = initial.copy()
-        self.x = initial.copy()           
+        self.x = initial.copy()    
+        
+        if f is None:
+            f = ZeroFunction()
+            
+            if step_size is None:
+                step_size = 1   
```

Comment:
Does this need to be set to 1 if `f` is passed in as `ZeroFunction` rather than None?

Reply:
Yes, good catch. Have changed this in the set_step_size function